spec/README.md

```diff
@@ -0,0 +1,100 @@
+### UTM Adapter Specification
+
+This document details at a high level the requirements for a UTM adapter that will be integrated in QGCS codebase, on the UTM Service provider side, the interfaces . The **Deliverables** detail specific outputs that will be developed as a part of this effort. 
+
+## Background
+UTM services or U-space services as it is called in the EU are a set of digital services to enable safe integration of drones with manned aviation that are regulated and will be verified and certified in the EU. Automated verification of the ability to conform to UTM standards forms a part of the UTM Service provider (USSP) certification framework. 
+
+The EU U-space law builds a regulatory framework for UTM, it is expected that other jurisdictions will have their own version of UTM regulations, while this work at this time (June 2023) uses language and services as described / used in the EU U-Space law, it is not limited to it. It is expected that the codebase will change as other jurisdictions will build a regulatory / legal framework for integrating drones in the airspace and enabling "advanced" operations like BVLOS etc. 
+
+### UTM Standards 
```

Comment:
Also:
- ED-270 - MOPS for UAS geocaging
- ED-282 - MOPS for UAS E-Reporting (something we probably will override with UTM SG work)
- UTM SURM SDSP (https://www.astm.org/workitem-wk69690)

Reply:
Thanks I have added it now.